VideoAPI/Video.API/Controllers/ConferenceController.cs

```diff
@@ -373,6 +374,17 @@ private async Task SafelyRemoveCourtRoomAsync(Guid conferenceId)
             }
         }
 
+        private async Task DeleteAudioRecordingApplication(Guid conferenceId)
+        {
+            var getConferenceByIdQuery = new GetConferenceByIdQuery(conferenceId);
```

Comment:
Don't think we need this extra check here Valentina. We can just use the 404 returned form the delete instead.

Reply:
We can remove this extra call

Reply:
I did not check here, I get HearingRefId to remove audio recording application, as it referenced to hearing Id, not conference Id